service/tablet_allocator.cc

```diff
@@ -2714,6 +2786,17 @@ class load_balancer {
             }
         });
 
+        // Invariant: node.capacity || node.drained
+        for (auto& [host, node] : nodes) {
+            if (node.drained) {
+                continue;
+            }
+            if (!node.capacity) {
+                lblogger.info("Cannot balance because capacity of node {} (or more) is unknown", host);
```

Comment:
Need to fall back to capacity = 1 if TABLET_LOAD_STATS_V2 is not enabled (rolling upgrade).